intl/hn/FAQ.md

```diff
@@ -0,0 +1,56 @@
+
+## सजिप्रा (अक्सर पूछे जाने वाले प्रश्न):
+  
+**क्यू:** मैं उन लोगों के साथ कैसे जुड़ सकता हूं जो इस चुनौती को उठाते हैं?
+
+**उत्तर:** 100DaysOfCode की अधिकृत वेबसाइट पर प्रस्तुत संसाधनों की सूची का उपयोग करने का सबसे अच्छा तरीका है: www.100DaysOfCode.com/connect
+  इसके अलावा, आप [100DaysOfCode Slack चैनल](https://join.slack.com/t/100xcode/sared_invite/zt-eivg7x1x-wgNPD77__4GcUwZNT8Zg) पर निमंत्रण लिंक का उपयोग करके स्लैक चैनल से जुड़ सकते हैं।
```

Comment:
Same here, the link is not right - I think it might be just because shared is spelled as sared, otherwise it looks like it's the same link so probably a typo: 

Use this one: https://join.slack.com/t/100xcode/shared_invite/zt-eivg7x1x-wgNPDh7ug_u4GcUwZNT8Zg

Reply:
Updated